robottelo/ui/locators.py

```diff
@@ -615,6 +615,10 @@ def __iter__(self):
         By.XPATH, "//a[contains(@ui-sref,'manifest.details')]"),
     "subs.import_history": (
         By.XPATH, "//a[contains(@ui-sref,'manifest.history')]"),
+    "subs.import_history.imported.success": (
+        By.XPATH, "//td[text()[contains(.,'imported successfully')]]"),
```

Comment:
nitpick: i think `//td[contains(.,'imported successfully')]` should work too, you can drop that `text()[` part.